src/GovUk.Education.ExploreEducationStatistics.Content.Api/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -976,13 +975,12 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                             Order = 0,
                             PublicationId = new Guid("cbbd299f-8297-44bc-92ac-558bcf51f8ad"),
                             Published = new DateTime(2018, 3, 22, 0, 0, 0, 0, DateTimeKind.Unspecified),
-                            ReleaseName = "2016 to 2017",
+                            ReleaseName = "2016",
```

Comment:
I dont think we should be changing this, the content has been user tested, and both API and UI tests will be looking for 2016 to 2017.

Reply:
It looks like this has changed - the releaseName is basically just the year and the actual name to display to the users is the time identifier and the year. According to design anyway.

Reply:
Spoken to Mark - sounds like there might be an outstanding ticket to change this to just year anyway. He is happy for this to get pushed in and to make any changes to broken tests

Reply:
I think we should probably grab laura and confirm this, all current publications on gov.uk follow the `A to B` format e.g. https://www.gov.uk/government/statistics/pupil-absence-in-schools-in-england-2014-to-2015 so i'm still a bit cautious of us making this change.

Reply:
changes now made to Freds requirement of e.g. 2019/20